drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -25,8 +25,10 @@
 #include "fsl_xcvr.h"
 
 #define KW41Z_DEFAULT_CHANNEL		26
-#define KW41Z_SEQ_SYNC_TIMEOUT		128
-#define KW41Z_ACK_WAIT_TIMEOUT		320
+#define KW41Z_CCA_TIMEOUT		8
+#define KW41Z_SHR_PHY_TIMEOUT		12
+#define KW41Z_PER_BYTE_TIMEOUT		2
```

Comment:
Is "timeout" really the best word here? I'd guess that just "time" conveys it better. This applies to all these defines of course.